vultr/resource_vultr_load_balancer.go

```diff
@@ -248,6 +253,15 @@ func resourceVultrLoadBalancerCreate(ctx context.Context, d *schema.ResourceData
 		instanceList = nil
 	}
 
+	var globalRegionsList []string
+	if attachGlobalRegions, globalRegionsOk := d.GetOk("global_regions"); globalRegionsOk {
+		for _, regionID := range attachGlobalRegions.(*schema.Set).List() {
```

Comment:
We should move away from passing copies around and stick with using the list index.

Reply:
```go
for i := range things
    thing = append(thing, things[i])
```